vms/platformvm/state/state.go

```diff
@@ -1486,11 +1488,11 @@ func (s *state) init(genesisBytes []byte) error {
 	return s.Commit()
 }
 
-func (s *state) AddStatelessBlock(block blocks.Block, status choices.Status) {
+func (s *state) AddStatelessBlock(block blocks.Block) {
```

Comment:
nit: should we err if the block status is not actually accepted?

Reply:
We don't have that information here.